lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscoverySpec.hs

```diff
@@ -293,7 +295,9 @@ prop_genChangeGap
 prop_genChangeGap g =
     property prop
   where
-    key = unsafeGenerateKeyFromSeed (mempty, mempty) mempty
+    Right phr =
+        fromText "abcdefghijklmnop" :: Either TextDecodingError (Passphrase "encryption")
+    key = unsafeGenerateKeyFromSeed (coerce phr, mempty) mempty
```

Comment:
`coerce` isn't needed here and the `fromText` before is rather misleading (since we are generating a seed). 
We could probably get away with simply: `seed = Hash "0000000000000000"` which conveys the idea that it doesn't matter.

Reply:
fixed in https://github.com/input-output-hk/cardano-wallet/pull/381/commits/fc2b3c898714e6ce5c4b5117a9938fe538a05e85